crates/test-utils/src/lib.rs

```diff
@@ -5,32 +5,51 @@ pub mod messages;
 pub mod network;
 pub mod objects;
 pub mod transaction;
-
 use rand::{rngs::StdRng, SeedableRng};
 use sui_types::{
     base_types::SuiAddress,
     committee::Committee,
-    crypto::{get_key_pair_from_rng, KeyPair, KeypairTraits, PublicKeyBytes},
+    crypto::{
+        get_key_pair_from_rng, AccountKeyPair, AuthorityKeyPair, AuthorityPublicKeyBytes,
+        KeypairTraits,
+    },
+    signature_seed::SignatureSeed,
 };
 
 /// The size of the committee used for tests.
 pub const TEST_COMMITTEE_SIZE: usize = 4;
 
 /// Generate `COMMITTEE_SIZE` test cryptographic key pairs.
-pub fn test_keys() -> Vec<(SuiAddress, KeyPair)> {
+pub fn test_keys() -> Vec<(SuiAddress, AuthorityKeyPair)> {
     let mut rng = StdRng::from_seed([0; 32]);
     (0..TEST_COMMITTEE_SIZE)
         .map(|_| get_key_pair_from_rng(&mut rng))
         .collect()
 }
 
+/// Generate `COMMITTEE_SIZE` test cryptographic key pairs.
+pub fn test_account_keys() -> Vec<(SuiAddress, AccountKeyPair)> {
+    let mut vec = Vec::new();
```

Comment:
Separates account from authorities in tests. Unfortunately requires seeding as Ecdsa get_key_pair_from_rng is not deterministic.